C/common/include/management_client.h

```diff
@@ -117,6 +117,8 @@ class ManagementClient {
 								const std::string& event);
 		int 			validateDatapoints(std::string dp1, std::string dp2);
 		AssetTrackingTable	*getDeprecatedAssetTrackingTuples();
+		std::string		getAlertByKey(const std::string& key);
+		bool			raiseAlert(const std::string& payload);
```

Comment:
Better to pass key, message and urgency, maybe even with a default value for urgency.